ui/app/components/modals/modal.js

```diff
@@ -267,7 +269,31 @@ const MODALS = {
 
   CUSTOMIZE_GAS: {
     contents: [
```

Comment:
I guess we also need to switch over the modal used on the send screen at some point (but not as part of this PR).

- [ ] Create github issue for using new customize gas modal component on send screen.